frontend/src/components/common/__snapshots__/ErrorModal.test.tsx.snap

```diff
@@ -4,6 +4,12 @@ exports[`Error modal tests... renders correctly 1`] = `
 <div
   aria-hidden="true"
 >
+  <div
```

Comment:
hm... this seems unexpected.

Reply:
This is because now ErrorModal is rendered inside `<TestCommonWrapper>` - it wasn't before. We need the test wrapper because our new button needs the ThemeContext for its styling to work